src/main/java/com/minecolonies/coremod/proxy/CommonProxy.java

```diff
@@ -25,13 +29,24 @@
 import java.util.HashMap;
 import java.util.Map;
 
-
 /**
  * CommonProxy of the minecolonies mod (Server and Client).
  */
 public class CommonProxy implements IProxy
 {
     /**
+     * Constants used for Registering Entities and Eggs
+     */
+    /* default */ private static final int TRACKING_RANGE            = 256;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this line start at column 5. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AIndentationCheck)